extensions/qr-code-scanner/src/index.tsx

```diff
@@ -12,19 +15,26 @@ import { randomInt } from "crypto";
 
 function qrDecode(filepath: string, callback: (data: string | boolean) => void) {
   decodeImage(filepath, (err, image) => {
+    if (!image) {
+      popToRoot()
+      return;
+    }
     if (err) {
-      confirmAlert({title: 'Image decoder error...'})
-        .then(() => callback(false))
+      showToast(ToastStyle.Failure, 'Image decoder error...')
+        .then(() => {
+          return
```

Comment:
That `return` statement would need to come after the `showToast()` call.

Reply:
_Facepalm_ Thanks for your patience.